set-web/src/main/java/eu/europeana/set/web/service/controller/BaseRest.java

```diff
@@ -200,4 +199,18 @@ protected String serializeUserSet(LdProfiles profile, UserSet storedUserSet) thr
     public String getApiVersion() {
         return getConfiguration().getApiVersion();
     }
+    
+    /**
+     * For the open sets with minimal profile we set the value to -1 so that the total will not be serialized
+     * @param profile
+     * @param userSet
+     * @return user set with updated total field
+     */
+    protected UserSet setTotalField(LdProfiles profile, UserSet userSet) {
```

Comment:
method not needed, please inline the code in applyProfile method

Reply:
code of this method integrated in applyProfile method